src/CMake/VisItWindowsFunctions.cmake

```diff
@@ -0,0 +1,155 @@
+# Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
+# Project developers.  See the top-level LICENSE file for dates and other
+# details.  No copyright assignment is required to contribute to VisIt.
+
+#*****************************************************************************
```

Comment:
I think this file could use a preamble comment describing what its intended to contain. I don't think the name alone captures it.

Reply:
Are you suggesting more than "This file contains functions specific to VisIt on Windows platform"?